src/main/java/com/epam/aidial/core/controller/PublicationController.java

```diff
@@ -161,7 +162,9 @@ public Future<?> listPublishedResources() {
                     ListPublishedResourcesRequest request = ProxyUtil.convertToObject(body, ListPublishedResourcesRequest.class);
                     String bucketLocation = BlobStorageUtil.buildInitiatorBucket(context);
                     String bucket = encryptService.encrypt(bucketLocation);
-                    return vertx.executeBlocking(() -> publicationService.listPublishedResources(request, bucket, bucketLocation), false);
+                    PermissionsFetcher permissionsFetcher = PermissionsFetcher.of(context, accessService);
+                    return vertx.executeBlocking(() -> publicationService.listPublishedResources(
```

Comment:
same here, we can adjust metadata response with permissions here

Reply:
Done.